src/elements/table/_table.scss

```
@@ -0,0 +1,46 @@
+//Table - base style
+
+@mixin table {
+  .#{$prefix}table {
+    width: 100%;
+  }
+
+  .#{$prefix}table-header {
+    @include typeset('normal', 2);
+    background-color: map-get($colormap-base, gray-mid);
+  }
+
+  .#{$prefix}table-header--cell {
```

BEM naming should be ...
```suggestion
  .#{$prefix}table-header__cell {
```